tools/generate-helpers/pg-table-bindings/search_categories.go

```diff
@@ -0,0 +1,46 @@
+package main
```

Comment:
We should aim to sync the obj type and search category names. Then this can be dropped.